pkgs/by-name/nl/nlohmann_json_schema_validator/package.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, nlohmann_json
+, cmake
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "nlohmann_json_schema_validator";
+  version = "2.3.0";
+
+  src = fetchFromGitHub {
+    owner = "pboettch";
+    repo = "json-schema-validator";
+    rev = "${finalAttrs.version}";
+    hash = "sha256-Ybr5dNmjBBPTYPvgorJ6t2+zvAjxYQISWXJmgUVHBVE=";
+  };
+
+  buildInputs = [ nlohmann_json ];
+  nativeBuildInputs = [ cmake ];
+
+  meta = with lib; {
+    description = "JSON schema validator for JSON for Modern C++ ";
+    homepage = "https://github.com/pboettch/json-schema-validator";
+    license = licenses.mit;
+    platforms = platforms.all;
+    maintainers = with maintainers; [ br337 ];
```

Comment:
Same here please, sort the attributes alphabetically.

Reply:
This should be in the correct order now.